OLSF/libra
# Web monitor Autopay view

## Motivation

Adds auto pay setup view to the web monitor.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(yes)

## Test Plan

1. Start a new swarm without an auto pay batch for Alice and checks the Web Monitor:
- Auto Pay tab with a warning message:

![image](https://user-images.githubusercontent.com/10797037/121973293-54959180-cd53-11eb-837d-05fdf4d66aa7.png)

- Dash tab with an Auto Pay warning icon in the Node Health:

![image](https://user-images.githubusercontent.com/10797037/121973413-8c043e00-cd53-11eb-8fad-53fc771fdac9.png)

2. Set an auto pay batch for Alice and check Dash, and Auto Pay tabs:
![image](https://user-images.githubusercontent.com/10797037/121973526-da194180-cd53-11eb-9614-0a8bfa706bf0.png)

![image](https://user-images.githubusercontent.com/10797037/121973668-2cf2f900-cd54-11eb-87da-1408b5d16583.png)

![image](https://user-images.githubusercontent.com/10797037/121973545-e6050380-cd53-11eb-9980-287074d54668.png)
